src/find_libpython/__init__.py

```diff
@@ -131,6 +131,23 @@ def wrapper(*args, **kwds):
     return wrapper
 
 
+def _get_proc_library(basename=False):
```

Comment:
Adding function flags for something the caller can do is a code smell.
```suggestion
def _get_proc_library():
```